examples/combined_ODNP.py

```diff
@@ -169,6 +169,9 @@
         target_directory = os.path.getcwd()
         filename = "temp_ctrl.h5"
         DNP_data.hdf5_write(filename, directory=target_directory)
+        # this doesn't make any sense -- how
+        # are you trying to write the data
+        # before it has been measured??
```

Comment:
This doesn't make any sense -- how are you trying to write the data before it has been measured??